tests/selenium/pixel_tracking_test.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+import unittest
+import pbtest
+
+class pixelTrackingTesting(pbtest.PBSeleniumTest):
```

Comment:
Nit: Class names should start with a capital letter (whereas instances start with lower case).

```python
class PixelSharingTest(ptest.PBSeleniumTest):
```